server/controllers/message.controller.js

```diff
@@ -8,7 +8,7 @@ const Message = db.Message;
  */
 function load(req, res, next, id) {
     Message.findById(id)
-    .then((message) => {
+    .then((message) => { 
```

Comment:
Make sure your editor accepts the `.editorconfig` in this repo, and that you run `yarn beautify` before committing.